test/input_gen/recorder.py

```diff
@@ -175,8 +175,8 @@ def train_step(model, optimizer, loss_fn, initial_input, label, writer_fn, **kwa
 
         # loss = loss_fn(label, outp[-1])
         loss = []
-        if kwargs.get('multi_out', None) != None:
-            multi_out = kwargs.get('multi_out', [])
+        if kwargs.get("multi_out", None) != None:
```

Comment:
this is minor but single quotes is more pythonic :)

Reply:
I am using black [formatter](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings). We'd better have a unified formatter :)